sass/_theme_rst.sass

```diff
@@ -328,7 +328,7 @@
   // that output similar, but not quite the same nesting so this is really the best we can do.
   dl:not(.docutils)
     // In Sphinx 2.0+ most DLs no longer contain .docutils, needs tighter scoping using next line
-    &.class, &.data
+    &.class, &.data, &.type, &.function
```

Comment:
Annoying that there isn't just a superclass for all of these.

Reply:
Yes, that's why I missed these classes previously.
Could be something to feedback to Sphinx, they made some changes to which elements are used for generated documentation, but not everything has a useful class. Some are just `<em>` for instance, which makes solving issues like #647 more difficult